Element Failing to build on AWS EC2 (t2.micro). ErrorNo: 137

I have already build element locally on my PC.
I am trying to build on AWS EC2 too. 
Specs i am using are: t2.micro:1GB ram and storage of 8 GB.

Command: npm i //ran perfectly
Command npm run services:start //Failing and its killing the contracts.
                                    
ubuntu@ip:~/decentralized-identity/element$ sudo npm run services:start

> @transmute/element@ services:start /home/ubuntu/decentralized-identity/element
> lerna run services:start --stream

lerna notice cli v3.15.0
lerna info Executing command in 1 package: "npm run services:start"
@transmute/element-lib: > @transmute/element-lib@0.0.2-2 services:start /home/ubuntu/decentralized-identity/element/packages/element-lib
@transmute/element-lib: > ./scripts/start_services.sh
@transmute/element-lib: 🧙 Starting Element Services...
@transmute/element-lib: ✨ Starting GANACHE on 8545
@transmute/element-lib: > @transmute/element-lib@0.0.2-2 contracts:migrate:dev /home/ubuntu/decentralized-identity/element/packages/element-lib
@transmute/element-lib: > truffle migrate --network development --reset
@transmute/element-lib: Compiling your contracts...
@transmute/element-lib: ===========================
@transmute/element-lib: Killed
@transmute/element-lib: npm ERR! code ELIFECYCLE
@transmute/element-lib: npm ERR! errno 137
@transmute/element-lib: npm ERR! @transmute/element-lib@0.0.2-2 contracts:migrate:dev: `truffle migrate --network development --reset`
@transmute/element-lib: npm ERR! Exit status 137
@transmute/element-lib: npm ERR! 
@transmute/element-lib: npm ERR! Failed at the @transmute/element-lib@0.0.2-2 contracts:migrate:dev script.
@transmute/element-lib: npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
@transmute/element-lib: npm ERR! A complete log of this run can be found in:
@transmute/element-lib: npm ERR!     /home/ubuntu/.npm/_logs/2019-07-30T06_18_24_902Z-debug.log
@transmute/element-lib: lerna notice cli v3.15.0
@transmute/element-lib: lerna info Executing command in 1 package: "npm run env:create:local"
@transmute/element-lib: @transmute/element-api: > @transmute/element-api@0.0.2-2 env:create:local /home/ubuntu/decentralized-identity/element/packages/element-api
@transmute/element-lib: @transmute/element-api: > node ./scripts/env_to_json_config.js local ../../example.env ./local.runtimeconfig.json && ./scripts/update_runtimeconfig_contract.sh ./local.runtimeconfig.json
@transmute/element-lib: @transmute/element-api: 👷 Modifying  ./local.runtimeconfig.json
@transmute/element-lib: @transmute/element-api: ✅ element.ethereum.anchor_contract_address:  0x1DABA81D326Ae274d5b18111440a05cD9581b305
@transmute/element-lib: lerna success run Ran npm script 'env:create:local' in 1 package in 0.7s:
@transmute/element-lib: lerna success - @transmute/element-api
@transmute/element-lib: /home/ubuntu/decentralized-identity/element/packages/element-lib
@transmute/element-lib: ✨ Starting IPFS on 5001
lerna success run Ran npm script 'services:start' in 1 package in 16.2s:
lerna success - @transmute/element-lib

--- a/README.md
+++ b/README.md
@@ -16,6 +16,25 @@ See also [ion](https://github.com/decentralized-identity/ion), [sidetree](https:
 
 ## Getting Started
 
+#### If you're using EC2
+
+We recommend using
+- Ubuntu Server 18.04 LTS
+- a t2.small instance or equivalent (1 vCPU, 2GB of RAM and 8 GB of Disk)
+
+To setup, run the following commands:
+```
+# Update packaging tool
+sudo apt update
+# Install nvm
+curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.34.0/install.sh | bash
+. ~/.nvm/nvm.sh
+# Install node
+nvm install v10.16.0
+# Install other required dependencies
+sudo apt install python build-essential jq
+```
+
 #### Clone the repo:
 
 ```

